src/server.tsx

```diff
@@ -150,14 +163,20 @@ const buildEpicData = async (
         countryCode: targeting.countryCode,
     };
 
-    const moduleUrl = `${baseUrl}/${type === 'ARTICLE' ? 'epic.js' : 'liveblog-epic.js'}`;
+    const moduleUrl = (): string => {
+        if (variantWithTickerData.modulePath) {
+            return `${baseUrl}/${variantWithTickerData.modulePath}`;
+        } else {
+            return `${baseUrl}/${type === 'ARTICLE' ? 'epic.js' : 'liveblog-epic.js'}`;
```

Comment:
I think I'm missing something, this looks like everyone will see the control?

Reply:
If the `modulePath` field is present on the variant then we use that (line 168), otherwise we go with the control version of the component

Reply:
I'll have a go at making this function clearer :)

Reply:
Oh wait haha It's literally the line above

Reply:
I was confused by the ticker data in the name so I think I just mentally ignored it

Reply:
I've pushed up a more concise version!